### DISCUSSION TITLE
how to export or download  the picture of my modeling result ？

### DISCUSSION THREAD

**Discussion by LiuYX1998** [Open-Systems-Pharmacology/Forum#1166] [Questions & Problems]
I have complete my pbpk modeling，how to export or download  the picture of my modeling result ？
![image](https://user-images.githubusercontent.com/97145041/157790653-6aa765fc-3a5c-4538-8338-f8520ecc1f07.png)
good luck

**Comment by msevestre** [comment]
Right click on the picture and you'll see a bunch of options to export the image
